libdnf/dnf-cmd.c

```diff
@@ -2,21 +2,21 @@
  *
  * Copyright (C) 2010-2015 Richard Hughes <richard@hughsie.com>
  *
- * Licensed under the GNU General Public License Version 2
+ * Licensed under the GNU Lesser General Public License Version 2.1
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- *(at your option) any later version.
+ * This library is free software; you can redistribute it and/or
```

Comment:
This is not a library. It is an example program. Furthermore, it should probably be moved out of here and into a folder like `demo` or something.